frontend/src/components/home/Home.tsx

```diff
@@ -0,0 +1,59 @@
+import { Link, useNavigate } from 'react-router-dom';
+import { useAuth0 } from '@auth0/auth0-react';
+
```

Comment:
It's good practice to group your imports. Libraries together, same directories together, different directories together. It's easier to navigate while reviewing code that way.